app/controllers/employees_controller.rb

```diff
@@ -34,6 +34,9 @@ def update
     @employee = Employee.find(params[:id])
     if @employee.valid?
       @employee.update(employee_params)
+      if params["Employprojects"].present?
+        @employee.saveprojects(params["Employprojects"]["p"], @employee.id)
```

Comment:
In the Update action of employees_controller.rb, I am updating the changes of employee details and saving newly selected projects. Here 'saveprojects' is a method which I defined in Employee model 'employee.rb'.